packages/tradershub/src/components/OptionsAndMultiplersSection/OptionsAndMultipliersContent/OptionsAndMultipliersContent.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { useActiveTradingAccount } from '@deriv/api';
+import { Button, Text, useBreakpoint } from '@deriv/quill-design';
+import { optionsAndMultipliersContent } from '../../../constants/constants';
+import { getStaticUrl, getUrlBinaryBot, getUrlSmartTrader } from '../../../helpers/urls';
+import { TradingAccountCard } from '../../TradingAccountCard';
+
+type TShowButtonProps = Pick<typeof optionsAndMultipliersContent[number], 'isExternal' | 'redirect'>;
+
+type TLinkTitleProps = Pick<typeof optionsAndMultipliersContent[number], 'icon' | 'title'>;
+
+const LinkTitle: React.FC<TLinkTitleProps> = ({ icon, title }) => {
+    const handleClick = (
+        event:
+            | React.KeyboardEvent<HTMLButtonElement>
+            | React.MouseEvent<HTMLButtonElement, MouseEvent>
+            | React.MouseEvent<HTMLDivElement, MouseEvent>
+    ) => {
+        event.persist();
+        switch (title) {
+            case 'Deriv Trader':
+                window.open(getStaticUrl(`/dtrader`));
+                break;
+            case 'Deriv Bot':
+                window.open(getStaticUrl(`/dbot`));
+                break;
+            case 'SmartTrader':
+                window.open(getUrlSmartTrader());
+                break;
+            case 'Binary Bot':
+                window.open(getUrlBinaryBot());
+                break;
+            case 'Deriv GO':
+                window.open(getStaticUrl('/deriv-go'));
+                break;
+            default:
+                break;
+        }
+    };
+    return (
+        // Had to result to button instead of div because of sonarcloud
+        <button
```

Comment:
Should we using Quill if we have to go for button?

Reply:
This is a clickable icon. Initially it was a div